container-images/rocm-fedora-gfx8/Containerfile

```diff
@@ -0,0 +1,11 @@
+FROM registry.fedoraproject.org/fedora:latest
```

Comment:
Would prefer to tag :latest to a fixed fedora version and let renovate update it.